frontend/src/reducers/MessageCenter.ts

```diff
@@ -83,54 +83,28 @@ const Messages = (
 
       const groups = state.groups.map(group => {
         if (group.id === groupId) {
-          const existingMessage = group.messages.find(message => {
-            // Note, we don't include detail when determining same-ness, just the main content.  This is to avoid
-            // trivial detail differences (like a timestamp).  If changing this approach apply the same change below
-            // for message removal.
-            return message.content === content;
+
+          // remove the old message from the list
+          const filteredArray = _.filter(group.messages, message => {
+            return message.content !== content;
           });
 
           let newMessage: NotificationMessage;
 
-          if (existingMessage) {
-            // it is in the list already
-            const firstTriggered = existingMessage.firstTriggered
-              ? existingMessage.firstTriggered
-              : existingMessage.created;
-
-            newMessage = createMessage(
-              state.nextId,
-              existingMessage.content,
-              existingMessage.detail,
-              messageType,
-              existingMessage.count + 1,
-              existingMessage.show_notification,
-              new Date(),
-              existingMessage.showDetail,
-              firstTriggered
-            );
-
-            // remove the old message from the list
-            const filteredArray = _.remove(group.messages, message => {
-              return message.content !== content;
-            });
-
-            group = { ...group, messages: filteredArray.concat(newMessage) };
-          } else {
-            newMessage = createMessage(
-              state.nextId,
-              content,
-              detail,
-              messageType,
-              1,
-              showNotification,
-              new Date(),
-              false,
-              undefined
-            );
-            group = { ...group, messages: group.messages.concat(newMessage) };
-          }
-
+          newMessage = createMessage(
+            state.nextId,
+            content,
+            detail,
+            messageType,
+            1,
+            showNotification,
+            new Date(),
+            false,
+            undefined
```

Comment:
So, in contrast with the old code, this new revision is always using default values for some params, while the old code copied values from the old replica.

OK, let's look at the screen shot of the MessageCenter for the case when there is more than one replica:

![image](https://user-images.githubusercontent.com/86114326/180980051-0127ecbd-cb1a-44c6-b7ae-3b5c068c5832.png)

If when invoking `createMessage` we always:
* pass `1` to the `count` arg, we will never see the `X in Y minutes` text, because it won't be considered as a replica.
* pass `undefined` to the `firstTriggered` arg, we would lose the date of the first replica and the `X in Y minutes` text may become `X in a few seconds ago minutes` if it happens to be visible.
* pass `false` instead of `existingMessage.showDetail` to the `showDetail` arg, we may end-up collapsing details of the message (they gray textarea) programatically.
* pass `showNotification` rather than `existingMessage.show_notification` to the `showNotification` arg ... well... this one apparently does not impact the old behavior.

That was the detailed technical explanation.

But in short, what I want to say is that this changes are fixing #5340, although we are losing the replica counter. It sounds negligible, so I open the question about if we want to lose the replica counter.

Also, there is the (also negligible) side effect that the for the replicated message, the user may see its details being collapsed automatically, which may be annoying for replicas triggered by the auto-refresh.

Both changes in behavior are negligible to me. So, I can approve if you are sure that you want them.

Reply:
@israel-hdez yes, I see what you mean and it's true, I prefer not losing that features. Thanks for the heads up. The message center is something I learning now and I've never noticed that.

So, I updated the code, honoring the count and the firstTriggered field now. Also, this makes the test to pass withouth any modification to them. I also tested locally, the last message is kept.